servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -109,11 +109,15 @@ def get_job(id, **kwargs):
     job = response.json()
 
     failures = [
-        format_failure(name, m)
+        format_task_failure(name, m)
         for name, metadata in job.get('calls', {}).items() for m in metadata
         if m.get('failures') is not None
     ]
 
+    # if there are no tasks/subworkflows but there are errors, get them
```

Comment:
Not sure if this comment is describing what the line below is doing 🤔

Reply:
How so?  `not len(failures)` tests if the previous attempt to gather failures from `calls` worked and `job.get('failures')` tests if there are failures elsewhere in the metadata.  Suggestions for making this clearer?

Reply:
Ah, I got it now. I somehow misunderstand the `if there are no tasks/subworkflows` just now. It's pretty clear!